compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/cfa/CfaUtils.kt

```diff
@@ -120,10 +127,156 @@ class PropertyInitializationInfoCollector(private val localProperties: Set<FirPr
 
     private fun processVariableWithAssignment(
         dataForNode: PathAwarePropertyInitializationInfo,
-        symbol: FirPropertySymbol
+        symbol: FirPropertySymbol,
+        reset: Boolean = false,
```

Comment:
`reset` is very unclear name. Maybe rename it to something like `overwriteRange`?

Reply:
Right, thanks for the suggestion. Changed.